### PR TITLE
Update controllers.markdown

### PR DESCRIPTION
**Pull request by jds11111**
These sticks are at the top of recommended sticks here, but many people are reporting stability problems with the sticks. The manufacturer, Silabs is apparently aware, but a fix has not been found for about half a year now (see the github page linked). It does not seem like a good idea to recommend them yet.  (Disclaimer: I have one of these sticks, but have not yet implemented it because of these problems.)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @jds11111, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21209).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_docs/z-wave/controllers.markdown [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: "Extended instructions how to setup Z-Wave."
 
 You need to have a compatible Z-Wave stick or module installed. The following devices have been confirmed to work with Z-Wave JS:
 
-- 700 series controllers
+- 700 series controllers (*Warning*: As of 15 January 2022 [users are still reporting stability problems with these controllers](https://github.com/zwave-js/node-zwave-js/issues/3906).)
```
Comment (frenck) [inline source/_docs/z-wave/controllers.markdown:10]: We don't track bugs in our documentation.

**Comment by frenck** [comment]
Thanks for willing to contribute, however, we do not track third party bugs in our documentation.

**Comment by jds11111** [comment]
Right, that makes sense. But it seems problematic that the top recommended devices are not working, and people are returning them.

**Comment by frenck** [comment]
If the upstream project retracts its support, I would say it makes sense to remove it?

**Pull request closed without merging**